modules/contact-form/admin.php

```diff
@@ -918,6 +918,11 @@ function grunion_ajax_spam() {
 		$status_html .= '</span></a></li>';
 	}
 
+	// If unit tests are running, the output needs to be returned and not echoed.
+	if ( defined( 'DOING_JETPACK_UNIT_TEST' ) && DOING_JETPACK_UNIT_TEST ) {
```

Comment:
You can use the existing `TESTING_IN_JETPACK` constant for scenarios like this one.

Reply:
Thanks! I should've checked for this. Should be fixed now.